ui/src/App.jsx

```diff
@@ -14,6 +14,10 @@ import DetailView from './pages/DetailPage';
 if (process.env.REACT_APP_BASE_URL) {
   axios.defaults.baseURL = process.env.REACT_APP_BASE_URL;
 }
+// Hack to disable cache from disk
+axios.defaults.params = {
```

Comment:
@MrSupiri can you explain this?

Reply:
https://github.com/MrSupiri/rancher-logging-pipeline-plumber/issues/33 you mentioned it we need to refresh the page after an update, So checked what's going on with the chrome's network tab. So it turns out chrome caches multiple GET requests going to the same location (in this case k8s API endpoint). Adding a cache-control header also didn't work because the caching is taking place on the client side.

When I googled this was the only solution I found. this is basically adding `?t=<CURRENT_TIME_EPOCH>`  towards the end. because `t` always changes chrome doesn't cache it thinking it's requesting new data.

Article I referred - https://stackoverflow.com/questions/49263559/using-javascript-axios-fetch-can-you-disable-browser-cache

Reply:
are we not using the websocket?

Reply:
No, it uses only GET and POST requests for now